app/Http/Controllers/Admin/PostController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller as BaseController;
+use Illuminate\Http\Request;
+use App\Models\Post;
+use App\Models\Category;
+use Illuminate\Support\Facades\Auth;
+
+class PostController extends BaseController
+{
+	public function create()
+	{
+		$categories = Category::all();
+		
+		return view('admin.post.create', compact('categories'));
+	}
+	public function store(Request $request)
+	{
+		// Validater
+		$this->validate($request, $this->_createRules());
+
+		// Addtion thumb_image to server and saving information post to database
+		$name = $request->thumb_image->store('images/');
+		$thumb_image = $request['thumb_image'];
+
+		$post = $request->all();
+		$post['user_id'] = Auth::user()->id;
+		$post['thumb_image'] = $name;
+
+		try {
```

Comment:
why try catch at here ?

Reply:
does not try catch. Validation ở trên  là được rồi ?

Reply:
ý em là khi tạo nhưng Db bị lỗi thì sao cơ? Chứ không phải dữ liệu lỗi